### PR TITLE
[Jsoncpp] Update to 1.9.2

### PR DESCRIPTION
**Pull request by etri3600**
**Describe the pull request**

- What does your PR fix? Fixes issue #
Fix implicit int to float conversion error in clang 10.

- Which triplets are supported/not supported? Have you updated the CI baseline?
Same as before. Only the version is changed.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
Yes.

### CODE REVIEW

File: ports/jsoncpp/portfile.cmake [hunk 1]
```diff
@@ -2,8 +2,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
```
Comment (LilyWangL) [inline ports/jsoncpp/portfile.cmake]: `include(vcpkg_common_functions)` is deprecated. Please remove it.

**Review by LilyWangL — CHANGES_REQUESTED** [review]

File: ports/jsoncpp/portfile.cmake (hunk 1 above)
Comment (etri3600) [reply ports/jsoncpp/portfile.cmake]: I deleted it.

**Comment by LilyWangL** [comment]
Thanks for your updates! The failures on Windows are not related to this changes. It should not block to merge this PR.

**Comment by etri3600** [comment]
> 
> 
> Thanks for your updates! The failures on Windows are not related to this changes. It should not block to merge this PR.

Can I show detail log or get a advice to solve the problem on windows build?

**Comment by LilyWangL** [comment]
> > Thanks for your updates! The failures on Windows are not related to this changes. It should not block to merge this PR.
> 
> Can I show detail log or get a advice to solve the problem on windows build?

You can click **Details** to check the CI testing result and download the error log.

**Comment by etri3600** [comment]
> 
> 
> > > Thanks for your updates! The failures on Windows are not related to this changes. It should not block to merge this PR.
> > 
> > 
> > Can I show detail log or get a advice to solve the problem on windows build?
> 
> You can click **Details** to check the CI testing result and download the error log.

Jsoncpp uses deprecated class in example code. That occurs errors on arm-uwp and x64-uwp.

**Comment by vicroms** [comment]
I added a patch based on changes made to upstream `jsoncpp` that allows disabling the examples with `JSONCPP_WITH_EXAMPLE`.  This patch could probably be removed with the next release of `jsoncpp`.

**Comment by vicroms** [comment]
Thanks for the PR!

**Pull request merged**